client/styles/layout/test-details.scss

```diff
@@ -687,6 +687,29 @@
                                 }
                             }
                         }
+
+                        &:disabled {
+                            opacity: 0.5;
+
+                            &:hover {
+                                color: inherit;
+                                border-color: #e9e9e9;
+                                background-color: inherit;
+    
+                                &._green-icon {
+                                    .btn-icon{
+                                        background-color: $green-new;
+                                        border-color: $green-new;
+                                    }
+                                }
+                                &._red-icon {
+                                    .btn-icon {
+                                        background-color: $red-new;
+                                        border-color: $red-new;
+                                    }
+                                }
+                            }
```

Comment:
code duplication & it doesn't code `:focus`, for example

Reply:
fixed